public/app/plugins/datasource/elasticsearch/query_ctrl.ts

```diff
@@ -51,18 +52,17 @@ export class ElasticQueryCtrl extends QueryCtrl {
   }
 
   queryUpdated() {
-    // As Raw Data and Raw Document have the same request, we need to run refresh if they are updated
-    const isPossiblyRawDataSwitch = this.target.metrics.some(
-      (metric: any) => metric.type === 'raw_data' || metric.type === 'raw_document'
```

Comment:
Is this logic not needed anymore?

Reply:
no, this new code replaces it. It follows up on this https://github.com/grafana/grafana/pull/26233/files#r454992652